code/game/mecha/mecha_construction_paths.dm

```diff
@@ -288,81 +288,81 @@
-					 //17
-					 list("key"=/obj/item/wirecutters,
-					 		"backkey"=TOOL_SCREWDRIVER,
-					 		"desc"="The wiring is added."),
-					 //18
-					 list("key"=/obj/item/stack/cable_coil,
-					 		"backkey"=TOOL_SCREWDRIVER,
-					 		"desc"="The hydraulic systems are active."),
-					 //19
-					 list("key"=TOOL_SCREWDRIVER,
-					 		"backkey"=TOOL_WRENCH,
-					 		"desc"="The hydraulic systems are connected."),
-					 //20
-					 list("key"=TOOL_WRENCH,
-					 		"desc"="The hydraulic systems are disconnected.")
+		//2
+		list("key"=TOOL_WRENCH,
+							"backkey"=TOOL_CROWBAR,
+							"desc"="External armor is installed."),
+		//3
+		list("key"=/obj/item/mecha_parts/part/gygax_armour,
+							"backkey"=TOOL_WELDER,
+							"desc"="Internal armor is welded."),
+		//4
+		list("key"=TOOL_WELDER,
+							"backkey"=TOOL_WRENCH,
+							"desc"="Internal armor is wrenched."),
+		//5
+		list("key"=TOOL_WRENCH,
+							"backkey"=TOOL_CROWBAR,
+							"desc"="Internal armor is installed."),
+		//6
+		list("key" = /obj/item/stack/sheet/metal,
+							"backkey" = TOOL_SCREWDRIVER,
+							"desc" = "Mech power core is secured."),
+		//7
+		list("key" = TOOL_SCREWDRIVER,
+							"backkey" = TOOL_CROWBAR,
+							"desc" = "Mech power core is installed."),
+		//8
+		list("key" = /obj/item/mecha_parts/core,
+							"backkey" = TOOL_SCREWDRIVER,
+							"desc" = "Mech power core is secured."),
+		//9
+		list("key"=TOOL_SCREWDRIVER,
+							"backkey"=TOOL_CROWBAR,
+							"desc"="Advanced scanner module is installed."),
+		//10
+		list("key"=/obj/item/stock_parts/scanning_module/adv,
+							"backkey"=TOOL_SCREWDRIVER,
+							"desc"="Scanning module is secured."),
+		//11
+		list("key"=TOOL_SCREWDRIVER,
+							"backkey"=TOOL_CROWBAR,
+							"desc"="Scanning module is installed."),
+		//12
+		list("key"=/obj/item/circuitboard/mecha/gygax/targeting,
+							"backkey"=TOOL_SCREWDRIVER,
+							"desc"="Peripherals control module is secured."),
+		//13
+		list("key"=TOOL_SCREWDRIVER,
+							"backkey"=TOOL_CROWBAR,
+							"desc"="Peripherals control module is installed."),
+		//14
+		list("key"=/obj/item/circuitboard/mecha/gygax/peripherals,
+							"backkey"=TOOL_SCREWDRIVER,
+							"desc"="Central control module is secured."),
+		//15
+		list("key"=TOOL_SCREWDRIVER,
+							"backkey"=TOOL_CROWBAR,
+							"desc"="Central control module is installed."),
+		//16
+		list("key"=/obj/item/circuitboard/mecha/gygax/main,
+							"backkey"=TOOL_SCREWDRIVER,
+							"desc"="The wiring is adjusted."),
+		//17
+		list("key"=/obj/item/wirecutters,
+							"backkey"=TOOL_SCREWDRIVER,
+							"desc"="The wiring is added."),
+		//18
+		list("key"=/obj/item/stack/cable_coil,
+							"backkey"=TOOL_SCREWDRIVER,
+							"desc"="The hydraulic systems are active."),
+		//19
+		list("key"=TOOL_SCREWDRIVER,
+							"backkey"=TOOL_WRENCH,
+							"desc"="The hydraulic systems are connected."),
+		//20
+		list("key"=TOOL_WRENCH,
+							"desc"="The hydraulic systems are disconnected.")
```

Comment:
could you go throught and clean up the spacing on these? What I mean is making sure they're not as far apart since a lot of the incorrect spacing was removed but not the tabs

Reply:
Fixed all this up across the file.